qa/tasks/cephfs/test_admin.py

```diff
@@ -194,6 +195,58 @@ def test_fs_new_pool_application_metadata(self):
             self._check_pool_application_metadata_key_value(
                 pool_names[i], 'cephfs', keys[i], fs_name)
 
+    def test_fs_rename(self):
```

Comment:
Add some more tests:

- renaming a fs that does not exist
- idempotency of rename
- renaming to a fs when an existing fs has that name already

Reply:
Add the 2 tests. I'm not sure what you mean by idempotency test here, successive rename operations with same argruments will fail, right?

Reply:
We could not throw an error if the target fs exists but the src does not.

Reply:
OK. You want a test to check whether `fs rename` fails if src fs name doesn't exist? Let me add that

Reply:
Thanks for the explanation!